spec/features/applications/show_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe "Application Show Page" do
   before(:each) do
     @application_1 = Application.create(name: "John", street_address: "1234 ABC Lane", city: "Turing", state: "Backend", zipcode: "54321", description: "I love animals", status: "In Progress")
-    @application_with_no_pets = Application.create(name: "John", street_address: "1234 ABC Lane", city: "Turing", state: "Backend", zipcode: "54321", description: "I love animals", status: "In Progress")
+    @application_with_no_pets = Application.create(name: "John", street_address: "1234 ABC Lane", city: "Turing", state: "Backend", zipcode: "54321", description: "I love animals", status: "In Progress") # do we want to hardcode the status in this spec or rely on the logic to update it? Tests failed when I removed it, but I'm not sure if thats because our logic is not working or if we need to populate it for the test.
```

Comment:
Aaah I see. Looking at this again, I think we can hard code it since its setting up test data, but we could also create @application_with_no_pets and then call @application_with_no_pets.set_status_in_progress. I like how you had it though, just another option!